docs/ui-components/price-breakdown.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: default
+title: Price Breakdown
+parent: UI Components
+nav_order: 4
+---
+
+# Price Breakdown
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+
+The price breakdown component displays information about Afterpay instalments and handles a number of common configurations.
```

Comment:
I think we have been using US English throughout this documentation, so for consistency we should use the US spelling of "installments" here.

```suggestion
The price breakdown component displays information about Afterpay installments and handles a number of common configurations.
```